include/rocksdb/convenience.h

```diff
@@ -16,6 +16,9 @@
 
 namespace ROCKSDB_NAMESPACE {
 class Env;
+class Logger;
```

Comment:
HISTORY.md needs to be updated, explain why we need to add registry to ConfigOptions, the benefit, how to use it.